recipes/scipyconference/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "scipyconference" %}
+{% set version = "0.2" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/scipyconference-{{ version }}.tar.gz
+  sha256: 98312c24d71ba2e6f90e61b165ccbe5ef791809c1f8e4a828ea6771904a4e154
+
+build:
+  entry_points:
+    - scipyconf = scipyconference.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=61.0
+    - wheel
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy
+
+test:
+  imports:
+    - scipyconference
+  requires:
+    - pip
+    - python {{ python_min }}
+
+about:
+  license: BSD-3-Clause
+  license_file: LICENSE
+  home: https://github.com/scipy-conference/puns-scipyconference
+  summary: SciPy-community-curated pun package
+
+extra:
+  recipe-maintainers:
+    - gforsyth
```

I can be another recipe wrangler 😸 
```suggestion
    - gforsyth
    - bsipocz
```